### ISSUE TITLE
Soundcloud add in Manager??

### ISSUE THREAD

**Issue by KaaaooS** [revancedapps/revancedmanager#25]
Its possible add Soundcloud?

**Comment by revancedapps** [comment]
Yes, adding Soundcloud to the Manager tool is possible, but on the APKMirror site, Soundcloud is distributed in XAPK format. Therefore, automatically extracting and building Soundcloud is not as straightforward as with other apps. When I have time, I will integrate [APKEditor ](https://github.com/REAndroid/APKEditor)into the automatic build process, and tools for XAPK format will be supported.

**Comment by KaaaooS** [comment]
New notices for the Soundcloud on the manager??

**Comment by Kevin-Wong2016** [comment]
> New notices for the Soundcloud on the manager?? [@vancedapps](https://github.com/vancedapps)

As @vancedapps said in this [comment](https://github.com/vancedapps/rv-manager/issues/126#issuecomment-3364017345) @vancedapps won't be doing SoundCloud due to copyright issues, please patch the app yourself using the link provided.
